docs/installation/development-installation.md

```diff
@@ -0,0 +1,90 @@
+## Development Installation
+
+### Prerequisites
+
+* Java JDK 14 or later
+* Maven 3
+* NodeJs (16.9.1) and NPM (8.3.2) - [NodeJS Install](https://nodejs.org/en/download/package-manager/)
+* mariaDB 10.6 (available for development via docker-compose scripts)
+* using Keycloak is optional
```

Comment:
Is it? I mean there are a lot of roles and settings...

Reply:
for development - yes